backend/base/models.py

```diff
@@ -357,6 +357,10 @@ def clean(self):
         super().clean()
         if not self.timestamp:
             self.timestamp = datetime.now()
+        if self.type == "AA" or self.type == "BI" or type == "VE":
+            remark_instances = RemarkAtBuilding.objects.filter(building=self.building, student_on_tour=self.student_on_tour, type=self.type)
+            if remark_instances.count() == 1:
+                raise ValidationError("There already exists a remark of this type from this student on tour at this building.")
```

Comment:
* `from django.utils.translation import gettext_lazy as _`
* Put the string inside `_()`

More info: [wiki on i18n](https://github.com/SELab-2/Dr-Trottoir-4/wiki/i18n)

Reply:
I'll look later for a translation. You don't have to do it right now as git can't merge these translation files very well.